util/io/src/worker.rs

```diff
@@ -14,11 +14,13 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+use futures::future::{self, Loop};
 use std::sync::Arc;
 use std::thread::{JoinHandle, self};
 use std::sync::atomic::{AtomicBool, Ordering as AtomicOrdering};
 use deque;
 use service_mio::{HandlerId, IoChannel, IoContext};
+use tokio::{self};
```

Comment:
I guess it is safer to keep this `self` to be backward compatible but these are not needed since Rust 1.30